General_gNMI/04__Subscribe.robot

```diff
@@ -29,6 +29,14 @@ Subscribe ONCE with supported "encoding" values
     And subscription paths    ${GET-PATH}
     Then subscribe ONCE with supported encodings
 
+Subscribe ONCE does not aggregate by deault
```

Comment:
You have a typo in "default"; and if you fix that typo, you get a keyword name clash. (Btw. I did not have any objections against the original name here, it looked fine to me).